packages/SystemsThinkingWorkbench-Core.package/STWTransferConnection.class/instance/performSimulationStep.st

```diff
@@ -2,11 +2,11 @@ updating
 performSimulationStep
 	
 	| particle |
-	
-	self startNode level: self startNode level - self intake.
-	particle := STWParticle new center: (self position + self segment start); color: self startNode fillingMorph color.
-	self particles add: particle.
-	self addMorph: particle.
-	self lastStepTime ifNil: [
-		self startStepping.
-		self lastStepTime: DateAndTime millisecondClockValue.].
+	((self startNode level <= 0) and: (self startNode isNegativeAllowed not)) ifFalse: [
```

Comment:
Maybe refactor this to be true when the node can produce output.

Or maybe we can write nice code and add a `node>>canProduce: aNumber` and `node>>canReceive: aNumber`, which would be great.